src/main/resources/templates/athleteList.html

```diff
@@ -29,12 +29,11 @@
   </div>
 </nav>
 	<h1>Coach login Page</h1>
-	<!--
-	<p th:text="${'Welcome, ' + coachDTO + '!'}" />
```

Comment:
I'd remove this entirely.  Commented-out code should not be committed to version control.  1)  You can always go back in history to see what used to be there.  2) If you're trying an experimental feature, you can do it in a branch, and not impact master.